pymc/distributions/distribution.py

```diff
@@ -618,24 +618,24 @@ def dist(
 
 
 @singledispatch
-def _get_moment(op, rv, *rv_inputs) -> TensorVariable:
-    raise NotImplementedError(f"Variable {rv} of type {op} has no get_moment implementation.")
+def _moment(op, rv, *rv_inputs) -> TensorVariable:
+    raise NotImplementedError(f"Variable {rv} of type {op} has no moment implementation.")
```

Comment:
Just so I understand well, why exactly is it better/necessary to rename `_get_moment` to `_moment` and `get_moment` to `moment`? I feel like it's related to dispatching but I'm not familiar with this concept

Reply:
It's bot better, just a different name. The `get` feels a bit awkward in hindsight. Would be like having the `logp` and `logcdf` functions be called `get_logp`, `get_logcdf`...